dockerfile/cuda11.1.1.dockerfile

```diff
@@ -122,6 +122,18 @@ RUN echo PATH="$PATH" > /etc/environment && \
     echo LD_LIBRARY_PATH="$LD_LIBRARY_PATH" >> /etc/environment && \
     echo SB_MICRO_PATH="$SB_MICRO_PATH" >> /etc/environment
 
+#install STREAM
+RUN cd /tmp && \
+    wget https://download.amd.com/developer/eula/aocc-compiler/aocc-compiler-4.0.0_1_amd64.deb && \
```

Comment:
curious why we need aocc-compiler instead of original gcc/clang like example in https://www.cs.virginia.edu/stream/ref.html

Reply:
Right, this is to target AMD architecture. AMD recommended a specific executable for STREAM. We are building it like they recommended.